.github/scripts/download_assets.sh

```diff
@@ -50,9 +50,9 @@ if [ $# -gt 0 ]; then
     done
 else
     download smbert v0003
-    download smbert v0004
-    download smbert_mocked v0004
-    download e5_mocked v0000
-    download xaynia v0002
+    download smbert v0005
+    download smbert_mocked v0005
+    download e5_mocked v0001
+    download xaynia v0201
```

Comment:
:question: 201 not 2

Reply:
i discussed some time ago with @acrrd how we could version `xaynia` to indicate that it still refers to `xaynia 2` even if we have to do small updates to the archive. the easiest thing we came up with was to use the first two digits for the major and the last two digits for the minor version bump.

but since we for example now also have `ort v1.15.1` as an archive, we could do the same for `xaynia v2.1`. the downside would be that we lose the lexicographical ordering of `v0201`, `v0002`, etc.